examples/siro_sandbox/README.md

```diff
@@ -0,0 +1,136 @@
+# Sandbox Tool
+
+![siro_sandbox_screenshot](https://user-images.githubusercontent.com/6557808/230213487-f4812c2f-ec7f-4d68-9bbe-0b65687f769b.png)
+
+# Overview
```

Comment:
We need to provide data/install instructions before anyone can actually run these commands. I'm starting a Slack thread [here](https://cvmlp.slack.com/archives/C0460NTKM4G/p1697047920184969) because I'm not sure about the correct path forward.

Reply:
Ok, after some discussion in that thread, it looks like this will be a significant project to gather data to run the HITL tool in some way. For this PR, I suggest update `Examples commands` section with an `Example usage` section, that you can leave empty except for a todo ("installation steps and example terminal commands coming soon!").

Reply:
And see follow-up Asana task: https://app.asana.com/0/1205278804336687/1205701804903615